src/api-reference/user-provisioning/v4.spend-user-provisioning.md

```diff
@@ -0,0 +1,702 @@
+
+- [Products and Editions](#products-editions)
+- [Scope Usage](#scope-usage)
+- [Dependencies](#dependencies)
+- [Access Token Usage](#access-token-usage)
+- [`POST provisioning/v4/Bulk` endpoint for provisioning a new spend resource](#create-bulk-provision-post-request)
+- [`PATCH provisioning/v4/Bulk` endpoint for managing spend data](#create-bulk-provision-patch-request)
+- [`PUT provisioning/v4/Bulk` endpoint for managing spend data](#create-bulk-provision-put-request)
+- [`GET /spend/v4/Users` endpoint for retrieving all spend users in a company (paginated and filterable)](#get-users-endpoint)
+- [`GET /spend/v4/Users/{uuid}` endpoint for retrieving a user's specific aggregated spend information](#get-user-endpoint)
+
+- [Schemas](#schema)
+
+## <a name="products-editions"></a>Products and Editions
+
+- Concur Expense Professional Edition
+- Concur Expense Standard Edition
+- Concur Invoice Professional Edition
+- Concur Invoice Standard Edition
+- Concur Request Professional Edition
+- Concur Request Standard Edition
+
+## <a name="scope-usage"></a>Scope Usage
+
+| Name                           | Description                    | Endpoint                                   |
+| ------------------------------ | ------------------------------ | ------------------------------------------ |
+| `spend.user.general.read`      | View spend user information.   | `provisioning/v4/Bulk` & `/spend/v4/Users` |
+| `spend.user.general.writeonly` | Change spend user information. | `provisioning/v4/Bulk`                     |
+
+Please note that in addition to the spend scopes, you will need the scopes listed in core user provisioning in order to provision core user data.
+
+## <a name="dependencies"></a>Dependencies
+
+SAP Concur users must purchase Concur Expense in order to use the spend extensions to provision spend related data. This API only available to approved early access partners. Please contact your SAP Concur representative for more information.
+
+## <a name="access-token-usage"></a>Access Token Usage
+
+This API supports company level access tokens.
+
+## <a name="create-bulk-provision-post-request"></a>`POST provisioning/v4/Bulk` endpoint for provisioning spend data
+
+Creates one or more provisioning request containing spend relevant data using the `provision/v4/Bulk` endpoint. This section discusses the spend extensions and how to use them in tandem with a the core extensions to provision a user in SAP Concur with spend data. In order to create a user within concur, the provision request must contain the information required to also provision the core user within SAP Concur. The Spend User extension is the required foundation on which the other spend extensions depend. Without a successful Spend User save, the other spend extensions cannot succeed.
+
+POST is used to create a new resource within Concur.
```

Comment:
Question: Down here we talk about 'POST' and 'DELETE', but don't directly mention 'PATCH' or 'PUT'?

Reply:
I think maybe I can mention that we don't support delete at a higher level cause I think that its worth calling out but yeah this might not be the best spot cause it could be confusing.